src/v3/src/components/Form/LayoutContainer.tsx

```diff
@@ -32,19 +32,24 @@ const LayoutContainer: FunctionComponent<{ uischema: UISchemaLayout }> = ({ uisc
   const { type, elements } = uischema;
 
   const isHorizontalLayout = type === UISchemaLayoutType.HORIZONTAL;
-  const flexDirection = isHorizontalLayout ? 'row' : 'column';
+  const isHorizontalLayoutWithNoWrap = type === UISchemaLayoutType.HORIZONTAL_NOWRAP;
```

Comment:
If this is causing a wrapping problem, it may be better to not break apart the localization string and let us work on the correct implementation, meaning the entire text would just need to be the link (eg. `<a>Or set up Okta Verify on another mobile Device</a>`

Reply:
Nice, this is great, and we can still follow up with the jira you created to utilize the string without removing the `<$1>` tokens 🙏